do-release.sh

```diff
@@ -36,16 +36,22 @@ mkdir -p dist
 # see https://github.com/upx/upx/issues/222 -- UPX produces broken darwin executables.
 
 GOOS=linux GOARCH=arm GOARM=5 go build -o dstask -mod=vendor -ldflags="$LDFLAGS" cmd/dstask/main.go
+GOOS=linux GOARCH=arm GOARM=5 go build -o dstask-sync -mod=vendor -ldflags="$LDFLAGS" cmd/dstask-sync/main.go
```

Comment:
@naggie Are we ready to put this in the release process?

Reply:
Yes definitely! I think this is a killer plugin.